test/generate-async.test.ts

```diff
@@ -23,26 +23,4 @@ describe('generate-async', () => {
     await uno.generate('rule')
     expect(order).eql([1, 2])
   })
-
-  test('preflight-first', async () => {
```

Comment:
Why?

Reply:
Now parseToken first

Reply:
This will be a breaking change - Can we still keep the test but instead update the order?

Reply:
> This will be a breaking change - Can we still keep the test but instead update the order?

Yes, updated